finance/models.py

```diff
@@ -0,0 +1,145 @@
+from __future__ import unicode_literals
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+
+from candidate.models import SocialMediaModel, Person
+
+
+@python_2_unicode_compatible
+class Committee(SocialMediaModel):
+    COMMITTEE_TYPES = (
+        ('OF', 'Primarily-formed Official Committee'),
+        ('PF', 'Primariliy-formed Committee'),
+        ('IC', 'Independent Committee')
+    )
+    name = models.CharField(max_length=255)
+    address = models.ForeignKey('locality.Address', null=True)
+    locality = models.ForeignKey('locality.Locality', null=True)
+    committee_id = models.CharField(max_length=16, unique=True)
+    type = models.CharField(
+        max_length=2,
+        choices=COMMITTEE_TYPES,
+        help_text=''
+    )
+
+    def __str__(self):
+        return self.name
+
+
+@python_2_unicode_compatible
+class Company(SocialMediaModel):
```

Comment:
I'm not sure the product need for this model. I can imagine us building richer pages for corporations, but treating all contributors identically is all we're trying to do at the beginning, right?